cli/src/semgrep/app/scans.py

```diff
@@ -203,7 +203,7 @@ def start_scan(self, meta: Dict[str, Any]) -> None:
             logger.info(f"Would have sent POST request to create scan")
             return
 
-        logger.debug("Starting scan")
+        logger.debug(f"Starting scan: {json.dumps({'meta': meta}, indent=4)}")
```

Comment:
We'll likely want to mask this in the tests so the magicmock doesn't cause any regressions to the snapshots

Reply:
We don't snapshot the --debug version of the logs, so I think this is fine.